cmd/operator-sdk/build/cmd.go

```diff
@@ -91,8 +95,8 @@ func createBuildCommand(imageBuilder, context, dockerFile, image string, imageBu
 }
 
 func buildFunc(cmd *cobra.Command, args []string) error {
-	if len(args) != 1 {
-		return fmt.Errorf("command %s requires exactly one argument", cmd.CommandPath())
+	if len(args) != 1 && !skipImage {
```

Comment:
It shows not accurate, did you test locally?

Reply:
I think it is right, it certainly does what I want and allows the user to skip the image if they specify --skip-image. Logically it is identical to !(len(args) ==1 || skipImage) which might read better.

Reply:
You are right.

Reply:
Solved